File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-Java/features/DirectedWithEdges/GPL/Neighbor.java
package GPL;

import java.util.LinkedList;

// *************************************************************************
   
public class Neighbor {
    public  Vertex end;
    public  Edge edge;
        
    public Neighbor() {
        end = null;
        edge = null;
    }
        
    public Neighbor( Vertex v,  Edge e ) {
        end = v;
        edge = e;
    }

}

File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-CSharp/features/UndirectedWithEdges/Neighbor.cs
//layer Undirected;

using System.Collections;

// *************************************************************************
  
public class Neighbor : NeighborIfc
{
    public  Vertex end;
    public  Edge   edge;
        
    public Neighbor( )
    {
        end = null;
        edge = null;
    }
        
    public Neighbor( Vertex v,  Edge e )
    {
        end = v;
        edge = e;
    }

}
